test_framework/scripts/longhorn-setup.sh

```diff
@@ -393,11 +396,16 @@ main(){
       customize_longhorn_manifest_for_airgap
       install_longhorn_by_manifest "${TF_VAR_tf_workspace}/longhorn.yaml"
     elif [[ "${LONGHORN_INSTALL_METHOD}" == "helm-chart" ]]; then
-      create_registry_secret
+      if [[ "${CREATE_PRIVATE_REGISTRY_SECRET}" == false ]]; then
+        create_registry_secret
+      fi
```

Comment:
Do we still need this if clause? I think the secret will be handled in `customize_longhorn_chart_for_airgap` already?